benchmarks/nx-cugraph/pytest-based/run-gap-benchmarks.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# location to store datasets used for benchmarking
+export RAPIDS_DATASET_ROOT_DIR=/datasets/cugraph
+mkdir -p logs
+
+# list of algos, datasets, and back-ends to use in combinations
+algos="
+    pagerank
+    betweenness_centrality
+    louvain
+    shortest_path
+    weakly_connected_components
+    triangles
+    bfs_predecessors
+"
+datasets="
+   netscience
+   email_Eu_core
+   cit_patents
+   hollywood
+   soc-livejournal
+"
```

Comment:
Why is this repeated?

Reply:
Hi Don. That was removed once this PR was moved out of draft!